server/src/main/java/org/opensearch/node/resource/tracker/NodeResourceUsageTracker.java

```diff
@@ -52,6 +58,13 @@ public double getMemoryUtilizationPercent() {
         return 0.0;
     }
 
+    /**
+     * Return io stats average if we have enough datapoints, otherwise return 0
+     */
+    public IoUsageStats getIoUsageStats() {
+        return ioUsageTracker.getIoUsageStats();
+    }
+
     /**
      * Checks if all of the resource usage trackers are ready
      */
```

Comment:
Should we update isReady method ?

Reply:
We can do that but I am evaluating this in the `ioUsageTracker` class and returning `IoUsageStats` Object with 0.0 value as IoUsagePercent.